examples/weaviate/personalized_description_with_weaviate_and_gemini_api.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            model_id="embedding-gecko-001" # default model. You can switch to another model if desired
</pre><p> </p><p>Gecko is deprecated, can you use <code>embedding-001</code> or the text embedding model? From <a href="https://ai.google.dev/gemini-api/docs/models/gemini#text-embedding" target="_blank">here</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Updated to <code>embedding-001</code>!</p>